packages/compass-e2e-tests/tests/collection-rename.test.ts

```diff
@@ -84,7 +85,10 @@ describe('Collection Rename Modal', () => {
   before(async function () {
     compass = await init(this.test?.fullTitle());
     browser = compass.browser;
-    await browser.setupDefaultConnections();
+    // no need to setup connections if we are running against Atlas
+    if (!ATLAS_DOMAIN) {
+      await browser.setupDefaultConnections();
```

Comment:
I guess it's just temporarily only in this file? You'd need to do this in almost every test file and the pattern is to try to keep the test code as similar as possible (not to mention that changing this logic if we start adding things like that everywhere would be very hard)

Reply:
Well, in Atlas when you create new clusters, then the connections start showing up automatically in Data Explore. So there is no extra step needed to "setup" the connections via the connection creation modal.

So, I actually do think we would skip this step for every test *if* we're running against Atlas. Are there any concerns about this?

Reply:
If switch statements are generic like that, we add them in the helper methods themselves, not in the test case, so again, if this is just temporary because that's the only test case you are adding, that's fine, but for follow-up PRs we'd need to move this logic out of test cases

Reply:
Maybe I can add this check into the function itself? Would that be better?

Reply:
Oh oops, just saw you already suggested that lol